includes/Linker.php

```diff
@@ -228,6 +228,8 @@ public static function link(
 
 		# Note: we want the href attribute first, for prettiness.
 		$attribs = array( 'href' => self::linkUrl( $target, $query, $options ) );
+		Hooks::run( 'SetHrefForRedLink', array(&$attribs));
```

Comment:
There is no need to add a new hook, you can use `LinkEnd` (https://github.com/Wikia/app/pull/16414/files#diff-8ffbd08183616445af94793926721964R247)